<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>806</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="introNombre">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nº de ruedas</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nRuedas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="MotorCheck">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Potencia del motor</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPotencia">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="combustible">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Combustible</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tipoCombustible">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Electrico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gasolina</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diesel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hibrido</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Queroseno</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectColor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Blanco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="alas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>82</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reactores">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reactores</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="trenAterrizaje">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aterrizaje</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="locomotora">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Locomotora</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nVagones">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergencia</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pinchazo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rueda de Repuesto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kit de Repareción de Pinchazos</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrícula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generaMatricula">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nº de vagones:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="matricula">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>680</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>MotorCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBoxPotencia</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combustible</sender>
   <signal>clicked(bool)</signal>
   <receiver>tipoCombustible</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locomotora</sender>
   <signal>clicked(bool)</signal>
   <receiver>nVagones</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
